book/src/deploying.md

```diff
@@ -120,6 +120,9 @@ $ ./kubelet ...  --allow-privileged=true ...
 ## Enabling mount propagation
 Another feature that CSI depends on is mount propagation.  It allows the sharing of volumes mounted by one container with other containers in the same pod, or even to other pods on the same node.  For mount propagation to work, the Docker daemon for the cluster must allow shared mounts. See the [mount propagation docs][mount-propagation-docs] to find out how to enable this feature for your cluster.  [This page][docker-shared-mount] explains how to check if shared mounts are enabled and how to configure Docker for shared mounts.
 
+[mount-propagation-docs]: https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation
+[docker-shared-mount]: https://docs.portworx.com/knowledgebase/shared-mount-propagation.html
```

Comment:
Oh...Sorry I copied from https://github.com/kubernetes-csi/docs/blob/master/book/src/troubleshooting.md.
So I never before looked https://docs.portworx.com/knowledgebase/shared-mount-propagation.html contents.

But if this link indicates how to configure Docker's shared mount propagation, I think below links can be introduced instead:
- https://kubernetes.io/docs/concepts/storage/volumes/#configuration
- https://www.ibm.com/support/knowledgecenter/ja/SSBS6K_2.1.0.3/troubleshoot/kubelet_fails.html#scene4
- https://success.docker.com/article/not-a-shared-mount-error

Reply:
@saad-ali I want to use https://kubernetes.io/docs/concepts/storage/volumes/#configuration link because the description is very simple, but what do you think about it?

Reply:
That sounds good to me

Reply:
@saad-ali Thanks, I modified linkes.